src/ensembl/src/shared/components/accordion/components/AccordionItemPanel.test.tsx

```diff
@@ -43,15 +43,15 @@ describe('AccordionItemPanel', () => {
       );
 
       expect(
-        wrapper
-          .find(AccordionItemPanel)
-          .find('div')
-          .hasClass('accordionPanelDefault')
-      ).toBe(true);
+        container
+          .querySelector('.accordionDefault')
+          ?.querySelector('div')
+          ?.querySelector('.accordionPanelDefault')
```

Comment:
Can't this be written with just one `querySelector` like this?

```.querySelector('.accordionDefault .accordionPanelDefault')```

Reply:
That's done